docs/reference/commandline/inspect.md

```diff
@@ -1,23 +1,25 @@
 ---
-title: "inspect"
-description: "The inspect command description and usage"
-keywords: ["inspect, container, json"]
+description: The inspect command description and usage
+keywords:
+- inspect, container, json
+menu:
+  main:
+    parent: smn_cli
+title: inspect
 ---
 
 # inspect
 
 ```markdown
-Usage:  docker inspect [OPTIONS] NAME|ID [NAME|ID...]
+Usage:  docker inspect [OPTIONS] CONTAINER|IMAGE|TASK [CONTAINER|IMAGE|TASK...]
 
-Return low-level information on one or multiple containers, images, volumes,
-networks, nodes, services, or tasks identified by name or ID.
```

Comment:
Document needs to be reverted as well (other than the metadata, and `{% raw %}`)